### PR TITLE
Create CODEOWNERS

### LINKED ISSUE #35: Add CODEOWNERS with primary maintainers
Reticketed from #31, we were discussing making clear who to ping for review, but perhaps better to use existing patterns: 

> Also unclear if adding who to request review from is required, as perhaps better to set CODEOWNERS as you do in gaia: https://github.com/cosmos/gaia/blob/main/.github/CODEOWNERS

### PR DESCRIPTION
**Pull request by dcwalk**
Closes #35

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Primary repo maintainers
+*       @ebuchman @shahankhatch @zmanian @hxrts
```
Comment (hxrts) [inline .github/CODEOWNERS:4]:
```suggestion
*       @ebuchman @zmanian @hxrts @mariapao
```

**Comment by hxrts** [comment]
can merge once this change has been made

**Comment by dcwalk** [comment]
made the change!

**Comment by hxrts** [comment]
ty!!

**Pull request merged**